### PR TITLE
Add "read only" for read only resource references

### PR DESCRIPTION
**Pull request by DeepakRajendranMsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@DeepakRajendranMsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
@DeepakRajendranMsft - If you see the previous specs there is a folder named swagger inside the ap-version folder. For example: `azure-rest-api-specs / arm-network / 2016-03-30 / swagger / network.json `
Can you please make sure that the format is preserved for the new spec with api-version 2016-06-01. The swagger folder is missing over there.

**Comment by DeepakRajendranMsft** [comment]
@amarzavery Addressed your comments

**Comment by DeepakRajendranMsft** [comment]
not sure why the travis job is failing between the commits. the only difference is that i am moving the files

**Comment by amarzavery** [comment]
@DeepakRajendranMsft - 
```
Azure Swagger Schema Validation /home/travis/build/Azure/azure-rest-api-specs/arm-network/2016-06-01/swagger/network.json should be valid Swagger:
     Uncaught SyntaxError: Unexpected token ]
      at Object.parse (native)
      at test/test.js:53:31
      at fs.js:334:14
      at FSReqWrap.oncomplete (fs.js:95:15)
```

**Comment by amarzavery** [comment]
@DeepakRajendranMsft - There is an unnecessary comma after the last item in the enum array at line 6912 (EffectiveRoute.source)and 6924 (EffectiveRoute.state).

File: arm-network/2016-06-01/swagger/network.json [hunk 1]
```diff
@@ -5581,27 +5581,31 @@
     "FrontendIPConfigurationPropertiesFormat": {
       "properties": {
         "inboundNatRules": {
+          "readOnly": true,
           "type": "array",
           "items": {
             "$ref": "#/definitions/SubResource"
           },
           "description": "Read only.Inbound rules URIs that use this frontend IP"
```
Comment (FrankSiegemund) [inline arm-network/2016-06-01/swagger/network.json]: Space. This also impacts all other occurrences.

File: arm-network/2016-06-01/swagger/network.json [hunk 2]
```diff
@@ -5581,27 +5581,31 @@
...
         },
         "inboundNatPools": {
+          "readOnly": true,
           "type": "array",
           "items": {
             "$ref": "#/definitions/SubResource"
           },
           "description": "Read only.Inbound pools URIs that use this frontend IP"
```
Comment (FrankSiegemund) [inline arm-network/2016-06-01/swagger/network.json]: Same. Space before 'Inbound'.

File: arm-network/2016-06-01/swagger/network.json [hunk 3]
```diff
@@ -5581,27 +5581,31 @@
...
         },
         "outboundNatRules": {
+          "readOnly": true,
           "type": "array",
           "items": {
             "$ref": "#/definitions/SubResource"
           },
           "description": "Read only.Outbound rules URIs that use this frontend IP"
```
Comment (FrankSiegemund) [inline arm-network/2016-06-01/swagger/network.json]: Dito.

**Pull request merged**